kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorNetworkIT.java

```diff
@@ -0,0 +1,334 @@
+import com.amazonaws.auth.policy.actions.S3Actions;
+import com.amazonaws.auth.policy.resources.S3ObjectResource;
+import com.amazonaws.services.s3.model.CreateBucketRequest;
+import com.amazonaws.services.s3.model.ListVersionsRequest;
+import com.amazonaws.services.s3.model.ObjectListing;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import com.amazonaws.services.s3.model.S3VersionSummary;
+import com.amazonaws.services.s3.model.VersionListing;
+import io.confluent.common.utils.IntegrationTest;
+import io.confluent.connect.s3.S3SinkConnector;
+import io.confluent.connect.s3.S3SinkConnectorConfig;
+import io.confluent.connect.s3.format.json.JsonFormat;
+import io.confluent.connect.s3.storage.S3Storage;
+import io.confluent.connect.storage.partitioner.DefaultPartitioner;
+import io.confluent.testcontainers.squid.SquidProxy;
+import org.apache.kafka.connect.runtime.SinkConnectorConfig;
+import org.apache.kafka.connect.storage.StringConverter;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static io.confluent.connect.s3.S3SinkConnectorConfig.PART_SIZE_CONFIG;
+import static io.confluent.connect.s3.S3SinkConnectorConfig.REGION_CONFIG;
+import static io.confluent.connect.s3.S3SinkConnectorConfig.S3_BUCKET_CONFIG;
+import static io.confluent.connect.storage.StorageSinkConnectorConfig.FLUSH_SIZE_CONFIG;
+import static io.confluent.connect.storage.StorageSinkConnectorConfig.FORMAT_CLASS_CONFIG;
+import static io.confluent.connect.storage.partitioner.PartitionerConfig.PARTITIONER_CLASS_CONFIG;
+import static org.apache.kafka.connect.runtime.ConnectorConfig.*;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Integration test for S3 Sink Connector.<p>
+ * Refer file /bin/README.md to find prerequisites
+ * to run the following tests.
+ */
+@Category(IntegrationTest.class)
+public class S3SinkConnectorNetworkIT extends BaseConnectorNetworkIT {
+
+  private static final Logger log = LoggerFactory.getLogger(S3SinkConnectorIT.class);
+
+  private static final String CONNECTOR_NAME = "s3-sink-connector";
+  private static final String STORAGE_CLASS_CONFIG = "storage.class";
+  private static final int NUM_RECORDS_PRODUCED = 1000;
+  private static final int TASKS_MAX = 1;
+  private static final String TEST_TOPIC_NAME = "topic1";
+  private static final String EXPECTED_FILE_EXTENSION = "json";
+  private static final List<String> KAFKA_TOPICS = Arrays.asList(TEST_TOPIC_NAME);
+  private static final int FLUSH_SIZE = 200;
+  private static final int EXPECTED_PARTITION = 0;
+  private static SquidProxy squid;
+
+  @Before
+  public void initializeResources() {
+    createS3RootClient();
+    // create the test bucket
+    createS3Bucket(TEST_BUCKET_NAME);
```

Comment:
See comments at the `getConnectorProps` below.
```suggestion
    createS3Bucket(TEST_BUCKET_NAME);
    setupConnectorProps();
```